src/Core/System/Snippet/SnippetValidator.php

```diff
@@ -5,26 +5,25 @@
 use Shopware\Core\Framework\Log\Package;
 use Shopware\Core\System\Snippet\Files\GenericSnippetFile;
 use Shopware\Core\System\Snippet\Files\SnippetFileCollection;
+use Symfony\Component\HttpKernel\KernelInterface;
 
 #[Package('discovery')]
-class SnippetValidator implements SnippetValidatorInterface
+class SnippetValidator implements BundleSnippetValidatorInterface
```

Comment:
This is a break

Reply:
is it? :thinking:

Reply:
ups, overlooked that BundleSnippetValidatorInterface extends the SnippetValidatorInterface :D